### PR TITLE
Add new landing page strings

### PR DESCRIPTION
**Pull request by codemist**
This implements the new strings for the landing page sections:
- Hero
- How it works
- Pricing
- Use cases

### CODE REVIEW

File: en/app.ftl [hunk 1]
```diff
@@ -92,6 +92,60 @@ hero-image-copy-unique-html = <strong>Use a unique relay address</strong> for ev
...
+# Hero Section
+landing-hero-headline = Protect your real email address to help control your inbox
+landing-hero-body = { -brand-name-firefox-relay } email aliases protect your real email address from public view, automatically forwarding emails to your real inbox. 
+    Now you can receive only the emails you want in your inbox. Sign up with your ⁨{ -brand-name-firefox-account }⁩ to get started.
+
+# How it works section
+landing-how-it-works-headline = How it works
+landing-how-it-works-body = Share { -brand-name-relay } email aliases instead of your real email address to protect your email inbox, as well as your identity.
+landing-how-it-works-step-1-body-cta = Download the { -brand-name-relay } extension for { -brand-name-firefox }
+landing-how-it-works-step-1-body = to generate email aliases automatically wherever you enter your email address online.
```
Comment (peiying2) [inline en/app.ftl:106]: This string and the string should be combined as one string, not separated like this.

File: en/app.ftl [hunk 2]
```diff
@@ -92,6 +92,60 @@ hero-image-copy-unique-html = <strong>Use a unique relay address</strong> for ev
...
+# Do not change '@relay.mozmail.com'
+landing-how-it-works-step-2-body = As you browse, the { -brand-name-relay } icon will appear where sites ask for your email address. 
+    Select it to generate a new, random address that ends in @relay.mozmail.com.
+landing-how-it-works-step-3-body = Sign in to the { -brand-name-relay } dashboard to keep track of the aliases you’ve created. { -brand-name-relay } will forward messages to your email address. 
+    If an alias gets spam or unwanted messages, you can block all messages or even delete the alias, right from the dashboard.
+
+# Pricing section
+landing-pricing-headline = Limited-time only: Unlimited aliases for $1 per month
```
Comment (peiying2) [inline en/app.ftl:114]: Currency should be a variable, as markets expansion go beyond where $ is used.

**Review by peiying2 — CHANGES_REQUESTED** [review]
@codemist  A couple of common issues:
- Need to update the comment # or ## used for Section and string specific. here is a doc on [how to use comments]( https://projectfluent.org/fluent/guide/comments.html)
- Currency, right now it is hard coded. Needs to be in variable so the system detects the location and country where the payment comes from.

File: en/app.ftl [hunk 3]
```diff
@@ -92,6 +92,60 @@ hero-image-copy-unique-html = <strong>Use a unique relay address</strong> for ev
...
+landing-use-cases-social-networks = Social Networks
+landing-use-cases-social-networks-body = Want to better protect your online identity when using a social network? 
+    Trying to keep your real email from being tied to your presence on social media? Use a { -brand-name-relay } alias to log in and help protect yourself online.
+landing-use-cases-offline = Offline
+landing-use-cases-offline-body = Prefer to get email receipts, but want to avoid marketing spam too?
+    Use a { -brand-name-relay } alias instead of your email when getting a paperless receipt in a store. 
+    We’ll forward the receipt to your real email address, and if you start getting emails you don’t like, just turn off email forwarding.
+landing-use-cases-access-content = Access Content
+landing-use-cases-access-content-body = Want to sign up for that newsletter, but don’t trust the sender? 
+    Use an alias instead of your real address—if you start getting unwanted emails through that alias, the sender may have sold your alias to other marketers, or it might even have been leaked in a data breach.
```
Comment (peiying2) [inline en/app.ftl:142]: `address—if` - let's add spaces before and after the M dash.

File: en/app.ftl (hunk 1 above)
Comment (peiying2) [reply en/app.ftl:106]: If you have the screenshot/mockup of this page, that would be helpful with context too.

File: en/app.ftl [hunk 4]
```diff
@@ -92,6 +92,62 @@ hero-image-copy-unique-html = <strong>Use a unique relay address</strong> for ev
...
+# Do not change '@relay.mozmail.com'
+landing-how-it-works-step-2-body = As you browse, the { -brand-name-relay } icon will appear where sites ask for your email address. 
+    Select it to generate a new, random address that ends in @relay.mozmail.com.
+landing-how-it-works-step-3-body = Sign in to the { -brand-name-relay } dashboard to keep track of the aliases you’ve created. { -brand-name-relay } will forward messages to your email address. 
+    If an alias gets spam or unwanted messages, you can block all messages or even delete the alias, right from the dashboard.
+
+# Pricing section
+# Variables:
+#   $USD (string) - currency
+landing-pricing-headline = Limited-time only: Unlimited aliases for 1{ $usd } per month
```
[the remark this answers is not in the document]
Comment (peiying2) [reply en/app.ftl:116]: I wonder whether `1` should be variable too.

[the remark this answers is not in the document]
Comment (peiying2) [reply en/app.ftl:116]: Or maybe this way, `For limited-time only, unlimited aliases monthly: (variable code). Here is a doc on this type of issue: https://projectfluent.org/fluent/guide/variables.html

File: en/app.ftl [hunk 5]
```diff
@@ -92,14 +92,14 @@ hero-image-copy-unique-html = <strong>Use a unique relay address</strong> for ev
 hero-image-copy-protect-html = … so you can <strong>protect your real email</strong> from tracking and spam.
 hero-image-copy-control-html = Now <em>you’re in control</em> of what’s arriving to your inbox!
 
-# Home Page Version A
+## Home Page Version A
 
-# Hero Section
+## Hero Section
```
Comment (codemist) [inline en/app.ftl:97]: @peiying2  I followed the document that you sent me, let me know if this is the right way to go about writing comments.

File: en/app.ftl (hunk 3 above)
Comment (codemist) [reply en/app.ftl]: Done!

File: en/app.ftl (hunk 1 above)
Comment (codemist) [reply en/app.ftl]: I seperated it into two now.

File: en/app.ftl (hunk 5 above)
Comment (peiying2) [reply en/app.ftl:97]: > @peiying2 I followed the document that you sent me, let me know if this is the right way to go about writing comments.

Check the doc's examples: 
- Double-hash comments (##) are always standalone, meaning there should be line space between the comment and the start of the strings
- For string specific # comment, it is immediately above the string. 

Please go through all the comments and fix those you have missed.

File: en/app.ftl (hunk 4 above)
[the remark this answers is not in the document]
Comment (maxxcrawford) [reply en/app.ftl:116]: So, here's the context for this string: https://drive.google.com/file/d/1ydINSgkWKWPxHnKFAFwYbF36mVHaJljf/view?usp=sharing

Specifically, the pricing module will need to be dynamically set per locale, so we should def. pull out the `1` from the variable. I think we can/should ship the currency and amount in the same variable, in account for which order they're presented in. `$1` versus `1$`, etc.

**Pull request merged**